examples/protobuf/GenericHost/Server/Program.cs

```diff
@@ -35,12 +35,15 @@
             .Bind(hostContext.Configuration.GetSection("Server"))
             .Configure(options =>
             {
+                string certificatePath = Path.Combine(
+                    hostContext.HostingEnvironment.ContentRootPath,
+                    hostContext.Configuration.GetValue<string>("Certificate:File")!);
+
                 options.ServerAuthenticationOptions = new SslServerAuthenticationOptions
                 {
-                    ServerCertificate = new X509Certificate2(
-                        Path.Combine(
-                            hostContext.HostingEnvironment.ContentRootPath,
-                            hostContext.Configuration.GetValue<string>("Certificate:File")!))
+                    ServerCertificateContext = SslStreamCertificateContext.Create(
+                        X509CertificateLoader.LoadPkcs12FromFile(certificatePath, password: null),
+                        X509CertificateLoader.LoadPkcs12CollectionFromFile(certificatePath, password: null))
```

Comment:
I don't see the point of these additionalCertificates. Should this parameter be null instead?